### ISSUE TITLE
Eval stacked logits

### ISSUE THREAD

**Issue by 3lLobo** [3lLobo/kaggle#11]
During evaluation, stack the logits and apply sigmoid only once at the end of eval. Better performance?
